.dialyzer.ignore-warnings

```diff
@@ -39,26 +39,28 @@
-apps/evm/lib/evm/gas.ex:304: Function operation_cost/1 has no local return
-apps/evm/lib/evm/gas.ex:304: The call 'Elixir.EVM.Gas':operation_cost(__@1::any(),'nil','nil','nil') breaks the contract (atom(),['Elixir.EVM':val()],['Elixir.EVM':val()],'Elixir.EVM.MachineState':t()) -> t() | 'nil'
-apps/evm/lib/evm/gas.ex:304: Function operation_cost/2 has no local return
-apps/evm/lib/evm/gas.ex:304: The call 'Elixir.EVM.Gas':operation_cost(__@1::any(),__@2::any(),'nil','nil') breaks the contract (atom(),['Elixir.EVM':val()],['Elixir.EVM':val()],'Elixir.EVM.MachineState':t()) -> t() | 'nil'
-apps/evm/lib/evm/gas.ex:304: Function operation_cost/3 has no local return
-apps/evm/lib/evm/gas.ex:304: The call 'Elixir.EVM.Gas':operation_cost(__@1::any(),__@2::any(),__@3::any(),'nil') breaks the contract (atom(),['Elixir.EVM':val()],['Elixir.EVM':val()],'Elixir.EVM.MachineState':t()) -> t() | 'nil'
-apps/evm/lib/evm/gas.ex:566: Function gas_cost_for_nested_operation/2 will never be called
+apps/evm/lib/evm/gas.ex:308: Function operation_cost/0 has no local return
+apps/evm/lib/evm/gas.ex:308: The call 'Elixir.EVM.Gas':operation_cost('nil','nil','nil','nil') breaks the contract (atom(),['Elixir.EVM':val()],['Elixir.EVM':val()],'Elixir.EVM.MachineState':t()) -> t() | 'nil'
+apps/evm/lib/evm/gas.ex:308: Function operation_cost/1 has no local return
+apps/evm/lib/evm/gas.ex:308: The call 'Elixir.EVM.Gas':operation_cost(__@1::any(),'nil','nil','nil') breaks the contract (atom(),['Elixir.EVM':val()],['Elixir.EVM':val()],'Elixir.EVM.MachineState':t()) -> t() | 'nil'
+apps/evm/lib/evm/gas.ex:308: Function operation_cost/2 has no local return
+apps/evm/lib/evm/gas.ex:308: The call 'Elixir.EVM.Gas':operation_cost(__@1::any(),__@2::any(),'nil','nil') breaks the contract (atom(),['Elixir.EVM':val()],['Elixir.EVM':val()],'Elixir.EVM.MachineState':t()) -> t() | 'nil'
+apps/evm/lib/evm/gas.ex:308: Function operation_cost/3 has no local return
+apps/evm/lib/evm/gas.ex:308: The call 'Elixir.EVM.Gas':operation_cost(__@1::any(),__@2::any(),__@3::any(),'nil') breaks the contract (atom(),['Elixir.EVM':val()],['Elixir.EVM':val()],'Elixir.EVM.MachineState':t()) -> t() | 'nil'
+apps/evm/lib/evm/gas.ex:573: Function gas_cost_for_nested_operation/2 will never be called
 apps/evm/lib/evm/machine_state.ex:53: Function subtract_gas/2 has no local return
 apps/evm/lib/evm/operation/environmental_information.ex:114: Function calldataload/2 has no local return
 apps/evm/lib/evm/operation/environmental_information.ex:117: The call 'Elixir.EVM.Helpers':decode_signed(binary()) breaks the contract (integer() | 'nil') -> 'Elixir.EVM':val() | 'nil'
 apps/evm/lib/evm/vm.ex:98: The pattern 'continue' can never match the type {'halt','invalid_instruction' | 'stack_underflow' | 'undefined_instruction'}
+apps/evm/lib/evm/operation/system.ex:24: Function create/2 has no local return
```

Comment:
I remember seeing these errors when @germsvel and I were pairing. @germsvel did we end up figuring out what causes these? We should probably try to avoid adding dialyzer warnings if we can 🙂

Reply:
I think it usually means that the function might raise an error. Note that it could be a function call that `create/2` is making within it that would raise the error, but the point remains. I think this is telling us that somewhere in `create/2` there's a possibility of an error being raised and so the function wouldn't have a local return. 

I agree that we should avoid adding dialyzer warnings if we can, especially when we're dealing with `evm` code. @ayrat555 did you try to figure out why this is now showing up when it wasn't before? Maybe the new code path added has the possibility of raising an error?

Reply:
Currently, I'm trying to figure out what causes this warning